### PR TITLE
incorporated feedback

### PR DESCRIPTION
**Pull request by rhoads-zach**
@lholmquist @lance let me know what you think

### CODE REVIEW

File: docs/topics/proc_deploying-the-configmap-booster-using-the-oc-cli-client.adoc [hunk 1]
```diff
@@ -132,7 +132,7 @@ ifdef::configmap-nodejs[]
 $ npm install && npm run openshift
 ----
 +
-These commands install any missing package dependencies, then using the xref:about-nodeshift[Nodeshift] package, launch the link:{link-s2i-process}[S2I process] on OpenShift to start the pod.
+These commands install any missing package dependencies, then using the xref:about-nodeshift[Nodeshift] package, deploy the booster on OpenShift.
```
Comment (lholmquist) [inline docs/topics/proc_deploying-the-configmap-booster-using-the-oc-cli-client.adoc]: can we change the word package to module

the other boosters will need this update to.  just tagging this one

Comment (rhoads-zach) [reply docs/topics/proc_deploying-the-configmap-booster-using-the-oc-cli-client.adoc]: nice catch, sorry i missed this.

Thank you for reviewing.

**Comment by rhoads-zach** [comment]
@lholmquist fixed

**Comment by lholmquist** [comment]
👍  merge away

**Comment by rhoads-zach** [comment]
thank you @lholmquist merging.

**Pull request merged**